deps/meck-0.7.2/src/meck.erl

```diff
@@ -71,7 +69,7 @@
 
 %% Records
 -record(state, {mod :: atom(),
-                expects :: dict(),
+                expects :: dict:dict(),
```

Comment:
We can update deps as we go. Seems out of scope and this change is pretty minimal.